netbeans/README.txt

```diff
@@ -0,0 +1,14 @@
+Steps to build and release the NetBeans Checker Framework plugin:
+1. Copy Checker Framework artifacts to the netbeans/CheckerFrameworkPlugin/release directory
+2. Open the CheckerFrameworkPlugin project in NetBeans
+3. Right click on the project in NetBeans and select "Create NBM"
+4. Copy the .nbm file created in step 3 from netbeans/CheckerFrameworkPlugin/build to netbeans/CheckerFrameworkPluginUpdateSite
+5. Update catalog.xml in netbeans/CheckerFrameworkPluginUpdateSite so that downloadsize matches the size of the .nbm file (in bytes)
+
+The above steps could also be done with ant tasks. There is a makenbm task included in the NetBeans build infrastructure that can be used for packaging the .nbm file.
```

Comment:
Do you mean the `nbm` task in checker-framework/netbeans/CheckerFrameworkPlugin/build.xml?  I would prefer command line instructs since those can easily be automated as part of the release process.